activiti-core/activiti-api-impl/activiti-api-process-runtime-impl/src/test/java/org/activiti/runtime/api/impl/ExtensionsVariablesMappingProviderTest.java

```diff
@@ -82,6 +86,142 @@ public class ExtensionsVariablesMappingProviderTest {
+            "�\u0015�W<�\u000B��y%z%ge�*�U�W\n" +
+            "�f��^\u0003�I^ӱ�dm���u����\u0013ק��-�>�4�݆�\n" +
+            "m\u0005�\u0005�\n" +
+            "F6\u0006ol(T.\u0014\u0017��乩�\u0007�\u000F�\u001Fz7;m\u07B3�[\u0011��j�}qY�-�-W\u007Ft�����[S������߆�&�v{����R�Ҽґ\u001D�v4��,��n��]�e�eU�������疷�1ݳmϗ\n" +
+            "~ŭJ�ʦ�z{7�����o`����*����?\t~�[\u001D\\�\\c^Sv\u0000{ �����\u0007�~f�\\\u007FH�P\uD946\uDFF5�ڡ�ȺK�n����\u000E�4�\n" +
+            "҆�#���\u001D\n" +
+            "8��h�Xݤ�T|\f\u001C�\u001E{�K�/���\u001D�8�8�x����S�SE�P���\u0016~�PkBk����\u001Dm�m�~�����љʳ\u001AgK�\u0011�\u0015��~>��D�����\u000B#\u001DK;\u001E\\��x�҂K��a�W.\u0007]����:\u007F��ʙn���W\u0019W[��^k�q�9���o�z]{���]o�s�k��\u007Fn�g�\u008D�\u001B�o�n^�5�V���w�,�3t�{�ٽ�{���ܟ|�n\u00103X�P�a�#�G5�[��4�:tv8`��q��\u0007#��\u0017\u007FH��2Z��������g��Ό\u0005��=_�|������?�������ɿ|��\u0019�\u001F\u001F}-~��͖�:ok�9�똈�x�>��䇢�:\u001F�>1>u}���tr�\u0017ܗ�_۾�}\u001B������-fO�\u0002(��SR\u0000xS\u000B\u00009\u0001\u0000\n" +
+            "�W\u0010\u0017N��S\u0016��w\u0000²�\u0001Y��\u007F�t�=��\u0015��u\u0000�\"U�\u0012��\u0005�\u001CaU�F \u001C�\u000B`''E\u0005�e:W���E\u0000\u001C�0Դ5���Ӧ�δ�y<2����\u001E�\n" +
+            "�]V���\u000F��\u0006�\n" +
+            "endstream\n" +
+            "endobj\n" +
+            "6 0 obj\n" +
+            "[ /ICCBased 7 0 R ]\n" +
+            "endobj\n" +
+            "2 0 obj\n" +
+            "<< /Type /Pages /MediaBox [0 0 595 842] /Count 1 /Kids [ 1 0 R ] >>\n" +
+            "endobj\n" +
+            "8 0 obj\n" +
+            "<< /Type /Catalog /Pages 2 0 R >>\n" +
+            "endobj\n" +
+            "9 0 obj\n" +
+            "<< /Title (Screenshot 2025-10-13 at 14.13.12) /Producer (macOS Version 15.6.1 \\(Build 24G90\\) Quartz PDFContext)\n" +
+            "/Creator (Preview) /CreationDate (D:20251013121425Z00'00') /ModDate (D:20251013121425Z00'00')\n" +
+            ">>\n" +
+            "endobj\n" +
+            "xref\n" +
+            "0 10\n" +
+            "0000000000 65535 f \n" +
+            "0000000171 00000 n \n" +
+            "0000013275 00000 n \n" +
+            "0000000022 00000 n \n" +
+            "0000000275 00000 n \n" +
+            "0000000364 00000 n \n" +
+            "0000013240 00000 n \n" +
+            "0000010386 00000 n \n" +
+            "0000013358 00000 n \n" +
+            "0000013407 00000 n \n" +
+            "trailer\n" +
+            "<< /Size 10 /Root 8 0 R /Info 9 0 R /ID [ <d146f1f990aa9ac7fa811ee2dced9e9b>\n" +
+            "<d146f1f990aa9ac7fa811ee2dced9e9b> ] >>\n" +
+            "startxref\n" +
+            "13632\n" +
+            "%%EOF\n");
+
+
+        //given
+        ExpressionResolver expressionResolver = mock(ExpressionResolver.class);
+        given(expressionResolver.containsExpression(any())).willReturn(false);
+        ReflectionTestUtils.setField(variablesMappingProvider, "expressionResolver", expressionResolver);
```

Comment:
Why using reflection for this? Couldn't the required expression resolver be passed directly via constructor?

Reply:
ops, this was an old test that I did to investigate a bug and I didn't notice that I pushed it (facepalm)